.github/workflows/reuse.yml

```diff
@@ -0,0 +1,15 @@
+name: REUSE Compliance Check
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: pull docker image
+      run: docker pull fsfe/reuse
+    - name: REUSE Compliance Check
+      run: docker run -v $(pwd):/lava-test-plans --workdir /lava-test-plans fsfe/reuse lint
```

Comment:
Can we change lava-test-plans to test-definitions or tmp?

Reply:
Done